### PR TITLE
Rename the "web" directory to "public" to become compatible with the updated recipe

### LINKED ISSUE #7: Flex recipe fails because public/check.php does not exist
When installing this library with Symfony Flex it fails, because [the recipe has been updated to replace `web` with `public`](https://github.com/symfony/recipes/commit/bf2148f9f1fe5af7e19c3145b6f7246c6cabb3a5#diff-43445c9887b945d546169a840318223e). However, this library still contains `web` and not `public`.

**Steps to reproduce**
- Create a new Symfony flex project (`composer create-project symfony/skeleton flex`)
- In the new directory, install the requirements checker (`composer require req-checker`)

**Expected result**
The requirements checker is installed and my `public` directory contains a file `check.php`.

**Actual result**
> Using version ^1.0 for symfony/requirements-checker
> ./composer.json has been updated
> Loading composer repositories with package information
> Updating dependencies (including require-dev)
> Package operations: 1 install, 0 updates, 0 removals
>   - Installing symfony/requirements-checker (v1.0.0): Downloading (100%)         
> Writing lock file
> Generating autoload files
> Symfony operations: 1 recipe
>   - Configuring symfony/requirements-checker (1.0): From github.com/symfony/recipes:master
> 
> Installation failed, reverting ./composer.json to its original content.
> 
>   [ErrorException]                                                                                                                      
>   copy(/home/nic/projects/flex/vendor/symfony/requirements-checker/public/check.php): failed to open stream: No such file or directory  
                                                                      

**Proposed solution**
Rename the `web` directory to `public`.

### PR DESCRIPTION
**Pull request by nicwortel**
This should resolve #7

### CODE REVIEW

File: src/SymfonyRequirements.php [hunk 1]
```diff
@@ -427,7 +427,7 @@ private function readComposer($rootDir)
             'etc-dir' => 'etc',
             'src-dir' => 'src',
             'var-dir' => 'var',
-            'web-dir' => 'web',
+            'web-dir' => 'public',
```
Comment (nicwortel) [inline src/SymfonyRequirements.php]: I'm not sure whether this key `web-dir` should also be changed to `public-dir`?

Comment (fabpot) [inline src/SymfonyRequirements.php]: yes, it should be renamed to `public-dir`

**Comment by fabpot** [comment]
Thank you @nicwortel.

File: src/SymfonyRequirements.php (hunk 1 above)
Comment (fabpot) [inline src/SymfonyRequirements.php]: done in 6abab31

**Pull request closed without merging**